flipperdevices/flipperzero-firmware
# Dolphin_srv timestamp and butthurt; Desktop_srv refactoring

# What's new
- Dolphin_srv: timestamp of last interaction added
- Leaving Flipper for one day without any icounter changes will result in butthurt increase
- Desktop debug screen updated to show last interaction timestamp and current level information
- Minor desktop_srv refactoring

# Verification 

- Build, flash
- Check debug info screen for details on current dolphin state

# Checklist (do not modify)
